src/discussions/posts/post/PostFooter.test.jsx

```diff
@@ -104,4 +92,24 @@ describe('PostFooter', () => {
     renderComponent({ ...mockPost, following: false });
     expect(screen.queryByRole('button', { name: /follow/i })).not.toBeInTheDocument();
   });
+
+  it('test like button when voteCount is zero', async () => {
+    renderComponent({ ...mockPost, voteCount: 0 });
+    expect(screen.queryByRole('button', { name: /like/i })).not.toBeInTheDocument();
+  });
+
+  it('test like button when voteCount is not zero', async () => {
```

Comment:
test('like button when voteCount is not zero', async () => {

please rewrite it like this.

Reply:
it and test represents same purpose, so here I' following the file's format

Reply:
change it to 
it('should test like button when voteCount is not zero', async () => {